A change in Qt 5.11_beta3 leads to errors

Arch GNU/Linux, `bionic` branch
```
CMake Error at src/downloads/qml/CMakeLists.txt:33 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```

--- a/src/downloads/qml/CMakeLists.txt
+++ b/src/downloads/qml/CMakeLists.txt
@@ -30,7 +30,7 @@ add_library(${TARGET} SHARED
 	${plugin_HDRS}
 )
 
-qt5_use_modules(${TARGET} Core Qml Quick)
+target_link_libraries(${TARGET} Qt5::Core Qt5::Qml Qt5::Quick)
 
 target_link_libraries(${TARGET}
 	${GLOG_LIBRARIES}

--- a/src/downloads/qml/CMakeLists.txt
+++ b/src/downloads/qml/CMakeLists.txt
@@ -30,6 +30,9 @@ add_library(${TARGET} SHARED
 	${plugin_HDRS}
 )
 
+find_package(Qt5Core REQUIRED)
+find_package(Qt5Qml REQUIRED)
+find_package(Qt5Quick REQUIRED)
 target_link_libraries(${TARGET} Qt5::Core Qt5::Qml Qt5::Quick)
 
 target_link_libraries(${TARGET}

